jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -13,6 +13,9 @@
     ></j-multiselect-toggle>
 
     <j-plugin-section-header style="margin-top: 12px">Viewers</j-plugin-section-header>
+    <v-row>
+      <span class="v-messages v-messages__message text--secondary">Export viewer plot as an image.</span>
+    </v-row>

```

Comment:
Looks like you missed the Tables section, I think that should also have a caption like this for consistency. Other than that, this PR looks good.